Tasks/ServiceFabricUpdateAppVersions/Update-ApplicationVersions.ps1

```diff
@@ -29,10 +29,13 @@ try {
 
     $logAllChanges = ((Get-VstsInput -Name logAllChanges -Require) -eq "true")
 
-    $versionSuffix = Get-VstsInput -Name versionSuffix -Require
-    Write-Host (Get-VstsLocString -Key VersionSuffixLabel -ArgumentList $versionSuffix)
+    $replaceVersion = ((Get-VstsInput -Name versionBehavior -Require) -eq "Replace")
+
+    $versionValue = Get-VstsInput -Name versionSuffix -Require
+    Write-Host (Get-VstsLocString -Key VersionValueLabel -ArgumentList $versionValue)
```

Comment:
This newVersion only applies to the application, but the versionValue applies to everything. The service and packages could have a different prefix. We log the newVersion on line 103 (line 100 before changes) anyways